xnmt/preproc.py

```diff
@@ -34,6 +35,8 @@ def from_spec(spec):
       for my_spec in spec:
         if my_spec["type"] == "lower":
```

Comment:
Just a comment: this specification of arguments via strings doesn't follow the current xnmt paradigm of specifying everything through classes, and could be fixed. In fact the whole preproc section might be worth refactoring. That can wait for another PR though.